utils/json.go

```diff
@@ -70,6 +58,10 @@ func GetAttributeAsString(obj map[string]interface{}, attr string) (string, erro
 // This is combination of samples for unmarshalling and linking
 // see: https://pkg.go.dev/github.com/ipld/go-ipld-prime
 func CIDFromJSONBytesUsingIpldPrime(content []byte) (string, error) {
```

Comment:
Maybe we should shorten the name of this to `CIDFromJSONBytes` now that the name is available.